src/EnergyPlus/PlantPipingSystemsManager.cc

```diff
@@ -2966,7 +2966,7 @@ namespace PlantPipingSystemsManager {
 
             // Create Y-direction partitions
 
-            CellWidth = this->VertInsThickness;
+            CellWidth = this->HorizInsThickness;
```

Comment:
@jcyuan2020 I was puzzled that the regression tests showed no diffs, but then I see that the test files which have horizontal insulation, also have vertical insulation using the same material (so the insulation thickness is the same for both). If @Myoldmopar agrees with this change, then go ahead an prepare a unit test.

Reply:
@mjwitte Thanks! The defect files would demonstrate the difference, where the different horizontal insulation thickness is used.

Reply:
I agree this change looks reasonable and would suggest modifying an example file to use a different horizontal insulation thickness as well as forging ahead on a unit test.

Reply:
Thanks! :thumbsup: